release-notes/docs/3.13.0.md

```diff
@@ -0,0 +1,85 @@
+# 3.13.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#7235](https://github.com/medic/cht-core/issues/7235): Make it possible to display previous task due date formatting via configuration
+
+
+## Highlights
+
+### Collect form data from external Android apps
+
+The CHT now supports forms that collect data from generic 3rd-party Android apps. Users can launch the apps from the form and then the output from the app is captured by the CHT and stored with the report. See the [CHT documentation](https://docs.communityhealthtoolkit.org/apps/reference/forms/app/#android-app-launcher) for more information on how to use this feature.
+
+> One example use case for this functionality is to integrate a COVID-19 configuration with a rapid diagnostic testing app (such as [RD-Toolkit](https://github.com/dimagi/rd-toolkit/)) to allow capturing the results of the RTD in the CHT report.
+
+This feature requires the CHT External App Launcher functionality included in [v0.10.0](https://github.com/medic/cht-android/releases/tag/v0.10.0) of cht-android. 
+
+- [#6981](https://github.com/medic/cht-core/issues/6981): Support collecting form data from external Android apps
+
+### Household Tasks Landing Page
+
+When the user has completed a task for a particular household and there are additional uncompleted task for that household, a landing page is shown with a list of the remaining tasks for that household. This allows the user to easily access all the tasks remaining for their current visit. 
+
+- [#5886](https://github.com/medic/cht-core/issues/5886): Make it easier to complete all tasks at a household in the same visit
+
+## And more...
+
+### Features
+
+- [#5886](https://github.com/medic/cht-core/issues/5886): Make it easier to complete all tasks at a household in the same visit
```

Comment:
Should this be listed in the `UI/UX changes` section?  (Not sure if that is just for "changes" or if it include totally new UI elements as well....)

Reply:
Yeah - I think this is correct @n-orlowski  - can you confirm this item [looks good here](https://github.com/medic/cht-core/blob/release_notes_3.13.0/release-notes/docs/3.13.0.md#uiux-changes)?

Reply:
Yep, this makes sense to me!